scripts/Convert-BpfToNative.ps1

```diff
@@ -3,6 +3,7 @@
 
 param([parameter(Mandatory = $true)] [string] $ProgramName,
     [parameter(Mandatory = $false)] [string] $SolutionDir = $Pwd,
+    [parameter(Mandatory = $false)] [string] $OutputDir = $Pwd,
```

Comment:
Need to chat about this. This will conflict with #1097

Reply:
Lets get this one in first, then we will fight with 1097.